app/services/ipam/db.rb

```diff
@@ -8,7 +8,7 @@ def suggest_ip
       to = subnet.to.present? ? IPAddr.new(subnet.to) : IPAddr.new(subnet_range.last.to_i - 2, subnet.family)
       (from..to).each do |address|
         ip = address.to_s
-        if !subnet.known_ips.include?(ip) && !excluded_ips.include?(ip)
+        if !subnet.known_ips.include?(ip) && !(excluded_ips || []).include?(ip)
```

Comment:
only internal db provider supports excluded_ips, but I could maybe define the attr_reader so it can never return nil which would be more robust in base class